python/lsst/ts/mtmount/mtmount_csc.py

```diff
@@ -2032,6 +2150,70 @@ async def handle_warning(self, reply):
             force_output=True,
         )
 
+    async def handle_apply_settings_set(
+        self, restore_defaults: bool, settings_to_apply: list[str]
+    ) -> None:
+        """Method to handle applying settings.
+
+        Parameters
+        ----------
+        restore_defaults : `bool`
+            Restore defaults before loading settings?
+        settings_to_apply : `list`[`str`]
+            Names of the settings to apply.
+        """
+        self.log.info("Starting background task to disable devices.")
+        self.should_be_commander = False
+        disable_devices_task = asyncio.create_task(self.disable_devices())
+
+        if restore_defaults:
+            self.log.info("Restoring default settings.")
+            await self.send_command(
+                commands.RestoreDefaultSettings(),
+                do_lock=True,
+                timeout=SHORT_COMMAND_TIMEOUT,
+            )
+
+        for setting in settings_to_apply:
+            self.log.info(f"Applying {setting=}.")
+            await self.send_command(
+                commands.ApplySettingsSet(settings=setting),
+                do_lock=True,
+                timeout=SHORT_COMMAND_TIMEOUT,
+            )
+
+        self.log.info("Waiting for devices to finish disabling.")
+        try:
+            await disable_devices_task
```

Comment:
Do you know how long it (more or less) takes to disable the devices? If yes, please add an ample timeout here to make sure that the CSC doesn't wait forever.

Reply:
So, this operation is done on a couple different places without a timeout. If I remember correctly, this is composed of a number of small operations and each one of them have a timeout already. So, I would not worry about this getting stuck forever.